inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfextract/PDFExtractor.java

```diff
@@ -221,6 +221,7 @@ else if (obj instanceof DrawOperator) {
 
     @Override public void drawImage(PDImage pdImage)
     {
+        throw new UnsupportedOperationException();
```

Comment:
Hm, I see why you are throwing a runtime exception here, but I'm not sure that it is a good idea. It puts the program into an unrecoverable state since the exception cannot be caught. Maybe logging would be the less aggressive solution.

Reply:
logging could be a less aggressive option but in this case, the test is about the image. So, to my understanding, I think it would be better to throw an exception.

Reply:
Which test? If the user opens a PDF with an image, then the pdfeditor fails horribly, instead of just not drawing the image.

Reply:
Ok, you convinced me. Actually, I read [this](https://airbrake.io/blog/java-exception-handling/unsupportedoperationexception) and it makes sense to throw an UnsupportedOperationException here, as calling the method is a programming error and we want to know when such an error happens.